src/Board.java

```diff
@@ -113,7 +147,7 @@ public boolean moveIsInCheck(Move move) {
         int srcColumn = move.src.column;
         Piece savedPiece = null;
 
-        if (movingPiece == null) return false;
+        if (movingPiece == null) return true;
```

Comment:
why does this switch to true? I'm sure there's a valid reason, just not sure when reading the code

(same goes for all the relevant bool values you flipped below)